providers/tutorial/manage.py

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env python3
+from plainbox.provider_manager import setup, N_
+
+setup(
+    name='com.canonical.certification:tutorial',
```

Comment:
All the other providers in our codebase have something like this:

```
    name='checkbox-provider-base',
    namespace='com.canonical.certification',
```

Is prefixing the `name` field with the `namespace` the way to run now?

Reply:
Interesting comment... I've tracked this down, and [found this](https://github.com/canonical/checkbox/blob/main/checkbox-ng/plainbox/impl/secure/providers/v1.py#L700C30-L700C30).
And I got even more interested in what this "_has_dedicated_namespace" is used for, and I found only one thing this affects is printing `namespace: {}` instead of `namespace: {} (derived from name)` when `./manage.py info` is run.


*slaps the roof of checkbox* this bad boy can fit so much over engineering in it.

Reply:
Let's proceed with having it separate so the `info` command prints only the unqualified (cleaner) part.